packages/dds/tree/src/test/cursorLegacy.spec.ts

```diff
@@ -0,0 +1,281 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+
+import { ITreeCursor, TreeNavigationResult } from "../forest";
+import { EmptyKey, FieldKey, JsonableTree } from "../tree";
+import { brand } from "../util";
+import { cursorTestCases } from "./cursor.spec";
+
+/**
+ * Tests the provided cursor factory with JsonableTree data. The cursor must be JSON compatible.
+ * @param suiteName - The name of the test suite to create.
```

Comment:
this is uses as part of the suiteName. Maybe "cursorName" or just "name" would be better?